client/api/src/client.rs

```diff
@@ -307,6 +307,8 @@ pub struct FinalityNotification<Block: BlockT> {
 	/// Finalized block header.
 	pub header: Block::Header,
 	/// Path from the old finalized to new finalized parent (implicitly finalized blocks).
+	///
+	/// This maps to the range `[old_finalized, new_finalized]`.
```

Comment:
FWIW I was also under the impression that this would already include the last finalized block.

Reply:
Last block is included in a separate explicit field to be more "inline" with the design of the import notification. This field contains from old-finalized (included) to new finalized (excluded).